content/en/cloud_cost_management/custom_allocation_rules/_index.md

```diff
@@ -0,0 +1,85 @@
+| Even  | Costs are allocated evenly to your destination tags. | Untagged support costs are allocated evenly to teams `teamA`, `teamB`, and `teamC`. |
+| Custom Percentage  | Costs are allocated based on custom percentages for the destination tags. | Untagged support costs are allocated 60% to `teamA`, 30% to `teamB`, and 10% to `teamC`. |
+
+## Allocation methods
+
+{{< tabs >}}
+{{% tab "Proportional Allocation" %}}
+
+Costs are allocated based on the proportional spend of destination values. Apply a filter to refine which part of the bill determines the proportions.
+
+{{< img src="cloud_cost/custom_allocation_rules/proportional_diagram.png" alt="Diagram illustrating the proportional split strategy" style="width:60%;" >}}
+
+{{< img src="cloud_cost/custom_allocation_rules/proportional_ui.png" alt="The proportional split strategy as seen in Datadog" style="width:60%;" >}}
+
+You can also specify how cost proportions should be partitioned to ensure segment-specific allocations. For example, if you partition your costs by `environment` using tags like `staging` and `production`, the proportions are calculated separately for each environment. This ensures allocations are based on the specific proportions within each partition.
+
+{{< img src="cloud_cost/custom_allocation_rules/proportional_partition_diagram.png" alt="Diagram illustrating the proportional split strategy with partitioning" style="width:90%;" >}}
+
+{{% /tab %}}
+
+{{% tab "Even Allocation" %}}
+
+With the even strategy, costs are allocated evenly towards your destination tags, regardless of any other spend. Similarly to proportional allocation, you can further customize your allocation by setting filters and partitions.
+
+{{< img src="cloud_cost/custom_allocation_rules/even_diagram.png" alt="Diagram illustrating the even split strategy" style="width:60%;" >}}
+
+{{< img src="cloud_cost/custom_allocation_rules/even_ui.png" alt="The even split strategy as seen in Datadog" style="width:60%;" >}}
+
+{{% /tab %}}
+
+{{% tab "Custom Percentage Allocation" %}}
+With the custom percentage strategy, you can define static custom percentages for the destination tags you select. For example, if you have 3 destinations (`teamA`, `teamB`, `teamC`) you can allocate 60% to `teamA`, 30% to `teamB`, and 10% to `teamC`.
+
+{{< img src="cloud_cost/custom_allocation_rules/custom_percentage_diagram.png" alt="Diagram illustrating the even split strategy" style="width:60%;" >}}
+
+{{< img src="cloud_cost/custom_allocation_rules/custom_percentage_ui.png" alt="The even split strategy as seen in Datadog" style="width:60%;" >}}
+
+{{% /tab %}}
+{{< /tabs >}}
+
+### Specify what costs are included in the allocation
+| Step | Required | Examples |
+| ---- | ---- | ---- |
+| Cost Provider | Yes | AWS, GCP, Azure |
```

Comment:
```suggestion
| Select the cost provider | Yes | AWS, Google Cloud, Azure |
```

Reply:
Suggesting to update the step's name to match how it's shown in the UI, which seems to be the pattern followed for other steps. This is also helpful for those using screen readers, as it can reduce ambiguity about what's being referenced